src/autoTune.ml

```diff
@@ -245,7 +245,7 @@ let isComparison = function
 
 let rec extractVar = function
   | UnOp (Neg, e, _) -> extractVar e
-  | Lval ((Var info),_) -> Some info
+  | Lval ((Var info),_) when not (List.exists (fun (Attr(s,_)) -> s = "goblint_stub") info.vattr) ->  Some info
```

Comment:
You may consider whether you want to extract the condition here and below into a function `is_goblint_stub`.